apps/crn-server/src/data-providers/types/discover.data-provider.types.ts

```diff
@@ -0,0 +1,5 @@
+import { DiscoverDataObject } from '@asap-hub/model';
+
+export type DiscoverDataProvider = {
```

Comment:
I was wondering if we don't want to add it here https://github.com/yldio/asap-hub/blob/master/packages/model/src/data-providers.ts#L53 :)

Reply:
I think these are the ones that are shared across the apps? Though to be fair I wouldn't mind having all the data provider types in the model package